echo-client.c

```diff
@@ -708,10 +678,13 @@ int main(int argc, char**argv)
 				goto out;
 			} else {
 				gettimeofday(&end_time, NULL);
-				timeval_diff(&start_time, &end_time,
-					     &diff_time);
-				sum_time += diff_time.tv_sec * 100000 +
-					diff_time.tv_usec;
+
+				timersub(&end_time, &start_time, &diff_time);
+
+				sum_time +=
+					(unsigned long long)diff_time.tv_sec *
+					100000ULL +
```

Comment:
Looks that one extra zero is missing here, this be multiplied by `1000000ULL` to get time in us.

Reply:
Indeed, good catch! Too many zeros there so it is easy to miss one :(